EventFilter/OnlineMetaDataRawToDigi/test/unpackMetaData.py

```diff
@@ -0,0 +1,50 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('TEST')
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('Configuration.StandardSequences.Reconstruction_cff')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+
+process.source = cms.Source(
+    "NewEventStreamFileReader",
+    fileNames = cms.untracked.vstring(
+        "file:/store/user/mommsen/testData/run302163_ls0787_streamExpress_StorageManager.dat"
```

Comment:
```file:``` doesn't seem appropriate here.
I wasn't able to find /store/user/mommsen/ at cern or fnal and the global redirector was not aware of it either.

I tried to run on the same run302163_ls0787 (not from streamExpress and using RAW file), but the outputs come out empty which seems like the data is not there and relevant methods in OnlineMetaDataRawToDigi::produce are not executed.
Please point me to a valid file location or send by email to be able to check at run time.

Reply:
The file is now available on EOS at CERN. Sorry, it took a while to get /store/user/mommsen/ set up. However, this file will also not trigger the unpacker, as it does not contain FED 1022. No data taken so far has this FED, see my comment from yesterday. I intend to replace the file once there is one which contains data from FED 1022.